src/styles/blocks/stars.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("../../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
It can be simplified
`background: url("../../images/star.svg") no-repeat center;`